app/views/historic_appointments/index.html.erb

```diff
@@ -1,49 +1,100 @@
 <% page_title "Past #{@role.name.pluralize}" %>
 <% page_class "historic-appointments historic-appointments-index govuk-width-container" %>
+
 <%= render "govuk_publishing_components/components/breadcrumbs", {
-    breadcrumbs: [
-      {
-        title: "Home",
-        url: "/"
-      },
-      {
-        title: "History of the UK Government",
-        url: "/government/history"
-      }
-    ]
-  } %>
-<header class="govuk-grid-row">
+  collapse_on_mobile: true,
+  breadcrumbs: [
+    {
+      title: "Home",
+      url: "/",
+    },
+    {
+      title: "How government works",
+      url: "/government/how-government-works",
+    },
+    {
+      title: "History of the UK government",
+      url: "/government/history",
+    },
+  ]
+} %>
+
+<div class="govuk-grid-row">
   <div class="govuk-grid-column-full">
     <%= render "govuk_publishing_components/components/title", {
-        context: "History",
-        title: "Past #{@role.name.pluralize}"
-      } %>
+      context: "History",
+      title: "Past #{@role.name.pluralize}",
+    } %>
   </div>
-</header>
+</div>
 
-<%= render partial: 'role_appointments_list', locals: { role: @role } %>
+<%# 21st century section %>
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-full">
+    <%= render "govuk_publishing_components/components/heading", {
+      text: "21st century",
+      padding: true,
+      border_top: 2,
+      font_size: "l"
+    } %>
+    <hr class="govuk-section-break govuk-section-break--m govuk-section-break--visible">
+  </div>
+</div>
 
-<div class="block featured-profiles">
-  <div class="inner-block floated-children">
-    <div class="year-block" id="modern-appointments">
-      <h2 class="profiles">20th &amp; 21st centuries</h2>
-      <ol>
-        <%= render partial: 'role_appointment', collection: @recent_appointments, locals: { role: @role } %>
-      </ol>
-    </div>
+<% @recent_appointments.in_groups_of(4, false) do |people| %>
+  <div class="govuk-grid-row historic-people-index__section-row-header" role="list">
```

Comment:
The use of `role="list"` and `role="listitem"` (in the partial) here is okay - just curious to why it isn't a `ul` or `ol`?

Reply:
I’m glad you flagged this one - in this context there needed to be various wrappers (ol > div > li) in order to wrap rows correctly inline with the [Design System Layout of `quarters`](https://design-system.service.gov.uk/styles/layout/#building-your-own-layout)

```
<div class="govuk-grid-row">
  <div class="govuk-grid-column-one-quarter">
    <p>govuk-grid-column-one-quarter</p>
  </div>
</div>
```

To apply this to the current layout using a `ol > li` markup is not possible. Consider:
```
<ol>
  <div class="govuk-grid-row">
    <li class="govuk-grid-column-one-quarter">
      <p>govuk-grid-column-one-quarter</p>
    </li>
    <li class="govuk-grid-column-one-quarter">
      <p>govuk-grid-column-one-quarter</p>
    </li>
  </div>
  <div class="govuk-grid-row">
    <li class="govuk-grid-column-one-quarter">
      <p>govuk-grid-column-one-quarter</p>
    </li>
  </div>
</ol>
```

I’ve been flip flopping between, using semantic markup this context & disregarding the Design System layout / grid and just replicating using bespoke CSS (or mimicking the Design system)

However if I do this this pages loses any Design System release especially that pertain to row / grid / layout.  This choice allows the page to keep Design System releases additionally gives more control over “what” is a list.  

I do think the vast major of `govuk-grid-column-one-quarter` uses will often will be a type of list - so in theory this is an use-case to raise on the Design System?

FWIW I agree - it feels a way. Good use of aria or an abomination? What’s your preference?

Reply:
Why so many grid rows? If you have one grid row and 9 quarters in it, they're going to wrap appropriately and be placed on 3 lines.

Can't we do eg:
<ol class="govuk-grid-row primeministers-c21">
    <li class="govuk-grid-column-one-quarter">
      <p>govuk-grid-column-one-quarter</p>
    </li>
    <li class="govuk-grid-column-one-quarter">
      <p>govuk-grid-column-one-quarter</p>
    </li>
    <li class="govuk-grid-column-one-quarter">
      <p>govuk-grid-column-one-quarter</p>
    </li>
</ol>

<ol class="govuk-grid-row primeministers-c20">
    <li class="govuk-grid-column-one-quarter">
      <p>govuk-grid-column-one-quarter</p>
    </li>
    <li class="govuk-grid-column-one-quarter">
      <p>govuk-grid-column-one-quarter</p>
    </li>
    <li class="govuk-grid-column-one-quarter">
      <p>govuk-grid-column-one-quarter</p>
    </li>
</ol>

Reply:
The example you've provided is a really good example of the problem. Every `grid-row` can hold 4 quarters (25% * 4) 

Those list items might be "20th Century" however it will be announced: 
ordered list, 3 items, 
List 1 of 3
Then
ordered list, 3 items etc.

When it really needs to be: 
ordered list 6 items.

Reply:
I disagree. When I look at that page I don't know there are 87 listed prime ministers without counting. I can get a rough idea of how many 20th century ones there are by doing x by 4 maths. The 21st century PMs are visually a separate list from the 20th c ones, the whole page is about all PMs. I don't think we necessarily need an over-arching container list for them all because those headings HRs,  etc interrupt it.

Reply:
Are you absolutely sure a grid row can only hold 4 quarters? I think it's unlimited and will wrap appropriately.

Reply:
Sorry, you disagree that it needs to be announced "per section" as one ordered list - so considering this Design:

![image](https://user-images.githubusercontent.com/71266765/130762118-59d43b19-57fb-427b-914b-ec62a6553de7.png)

Following this suggested markup - "20th Century" can be announced:
20th Century heading - ordered list 4 items, ordered list 4 items., ordered list 4 items. ordered list 4 items etc

Reply:
No, you have one grid row on the container ol, then however many 1/4 li inside it - it will wrap if you go over 4

Reply:
🤔  Maybe this is alluding to a different understanding of how layout within the Design System can be used.

@vanitabarrett I briefly reached out about this question the other day, can I ask a similar question echoing this:

> Are you absolutely sure a grid row can only hold 4 quarters? I think it's unlimited and will wrap appropriately.

Is it OK / correct to use unlimited `govuk-grid-column-one-quarter`s within one `govuk-grid-row`?  This would solve this issue.